test/unit/uk/gov/hmrc/apidocumentation/FiltersSpec.scala

```diff
@@ -20,7 +20,7 @@ import akka.actor.ActorSystem
 import akka.stream.{ActorMaterializer, Materializer}
 import org.mockito.Matchers.{any, eq => meq}
```

Comment:
Changing meq to eqTo makes the transition to mockito-scala easier.

Reply:
Nice - I'll do that